queries/sql/highlights.scm

```diff
@@ -27,13 +27,61 @@
 (term
   alias: (identifier) @variable)
 
+((term
+   value: (cast
+    name: (identifier) @function.call
+    parameter: [(literal)]?)))
+
 (literal) @string
 (comment) @comment
 (marginalia) @comment
 
 ((literal) @number
  (#lua-match? @number "^%d+$"))
 
+(parameter) @variable
```

Comment:
Can you tell us why `@parameter` doesn't apply here?

Reply:
Yes, you are right. I will change this to a `@parameter` this evening.